### PR TITLE
Page titles

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
##Summary
add branch page-titles

##Screenshot
captura de pantalla 2018-09-06 a la s 15 13 57

##Trello Card
https://trello.com/c/STWBkHIf/21-page-titles

### CODE REVIEW

File: scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.title-container{
+  display: flex;
+  margin:10px;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Add a space between `:` and `10`

File: scss/styles.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.title {
+  color: black;
+  font-size: 30px;
+  padding-top: 33px;
+  left:-45px;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: Add a space after `:`

File: scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.title-container{
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Add a space before `{`

File: index.html [hunk 4]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+<div class="title-container">
+  <img src="images/triangle.svg" width="120" height="90">
```
Comment (mmamaniwolox) [inline index.html]: For this, use a pseudo element like `&:before`

File: scss/styles.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.title {
```
Comment (mmamaniwolox) [inline scss/styles.scss]: This should be in `titles.scss` file

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+<div class="title-container">
```
Comment (mmamaniwolox) [inline index.html]: This should be tabbed, 2 more spaces to right, same with below element

File: scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+  color: black;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: Create a `colors.scss` file, import it in `application.scss` and use it

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: index.html [hunk 8]
```diff
@@ -9,6 +9,5 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
-</body>
+  <p class="title">Front-End Tools</p>
```
Comment (mmamaniwolox) [inline index.html]: As this is a title, you could use `<h1>` here

File: scss/color.scss [hunk 9]
```diff
@@ -0,0 +1 @@
+$color-black: #000000;
```
Comment (mmamaniwolox) [inline scss/color.scss]: This is a detail, but you can just write `#000` and will work the same. It works the same with repeated hexs, like `#FFF`

File: scss/titles.scss [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+.title {
+  color: $color-black;
+  font-size: 24px; 
+  position: relative;
+  left: 90px;
+  top: 60px;
+  z-index: 1;
+  
+
```
Comment (mmamaniwolox) [inline scss/titles.scss]: delete one of this spaces

File: scss/titles.scss [hunk 11]
```diff
@@ -0,0 +1,23 @@
+@import 'color';
+
+.title {
+  color: $color-black;
+  font-size: 24px; 
+  position: relative;
+  left: 90px;
+  top: 60px;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This is a relative position element, dont use left or top in this one. To move it use margin or padding. But dont focus on this in this PR, just remove them (top and left). We will see how to position the title once we start adding things

File: scss/titles.scss [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+  z-index: 1;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: This is not needed, you just have to use the z-index in the image

File: scss/titles.scss [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+  &:after {  
+  content: url(images/triangle.svg);
+  width: 90px;
+  height: 60px;
+  padding: 10px;
+  position: absolute;
+  display: inline-block;
+  top: -60px;
+  left: -80px;
```
Comment (mmamaniwolox) [inline scss/titles.scss]: Try to find a positive bottom and right position instead of negative top and left

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

File: scss/titles.scss [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+  position: relative;
+  margin: 80px;
+
+  &:after {  
+  content: url(images/triangle.svg);
+  width: 90px;
+  height: 60px;
+  padding: 10px;
+  position: absolute;
+  display: inline-block;
```
Comment (FrankIglesias) [inline scss/titles.scss]: this is unnecessary

File: scss/titles.scss [hunk 15]
```diff
@@ -0,0 +1,20 @@
+@import 'color';
+
+.title {
+  color: $color-black;
+  font-size: 24px; 
+  position: relative;
+  margin: 80px;
+
+  &:after {
```
Comment (FrankIglesias) [inline scss/titles.scss]: `&::after`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/titles.scss [hunk 16]
```diff
@@ -6,13 +6,12 @@
   position: relative;
   margin: 80px;
 
-  &:after {  
+  &:before {
```
Comment (FrankIglesias) [inline scss/titles.scss]: `&::before`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**